BitwardenShared/Core/Platform/Services/StateService.swift

```diff
@@ -1922,6 +1991,10 @@ actor DefaultStateService: StateService { // swiftlint:disable:this type_body_le
         .eraseToAnyPublisher()
     }
 
+    func trackUserAction(_ action: UserAction) async {
+        appSettingsStore.addUserAction(action)
```

Comment:
🤔 Shouldn't the action be added only if `appSettingsStore.reviewPromptShownForVersion != currentAppVersion`? Because if the review prompt has already been shown then it doesn't make sense to keep adding user actions; as this could grow with no limits.
However, if the idea is to keep counting the user actions, then I wouldn't store an array of `UserAction` but I would use an array of Tupple or of a simple struct like:
```swift
struct UserActionItem {
    let userAction: UserAction
    let count: Int
}
```
So then you just update the count each time a `UserAction` is tracked.

Reply:
@fedemkr that's a great question! We overlooked this scenario during our refinement session. We just discussed it in the stand-up, and we've decided that we will not continue tracking user actions once the app review prompt has been shown for the current version.